src/components/SwitchPlanPopup/SwitchPlanPopup.js

```diff
@@ -100,29 +109,21 @@ const SwitchPlanPopup = ({
               <SubscriptionIconStyled period={toOffer.period} showLabel="New" />
             </ImageWrapper>
             <TitleStyled step={step}>{t(toOffer.switchDirection)}</TitleStyled>
-            <TextStyled
-              step={step}
-              dangerouslySetInnerHTML={{
-                __html: `${t(`You are about to change your plan from <b>
-                  ${fromOffer.offerTitle}</b> to <b> 
-                  ${toOffer.title} </b>. You will be charged the new price <b>
-                  ${toOffer.nextPaymentPrice}${
-                  toOffer.nextPaymentPriceCurrencySymbol
-                } 
-                  </b> on your next billing date <b>
-                  ${dateFormat(fromOffer.expiresAt)}</b>.`)}
+            <TextStyled step={step}>
+              <Trans>
+                You are about to change your plan from <b>{{ fromOfferTitle }}</b> to <b>{{ toOfferTitle }}</b>. You will be charged the new price <b>{{ toNextPaymentPrice }}{{ toNextPaymentPriceCurrencySymbol }}</b> on your next billing date <b>{{ fromExpiresAt }}</b>.
+              </Trans>
+              <br />
+              { toOffer.couponNotApplicable && (
+                <>
                   <br />
-                  ${
-                    toOffer.couponNotApplicable
-                      ? `<br />
-                    ${t(
-                      'Your current coupon will not apply to the new plan. If you have a coupon for your new plan, you can apply it after confirming your switch.'
-                    )} <br />`
-                      : ''
-                  }
-                  <br /> ${t('Do you want to apply the change now?')}`
-              }}
-            />
+                  {t('Your current coupon will not apply to the new plan. If you have a coupon for your new plan, you can apply it after confirming your switch.')}
+                  <br />
+                </>
+              )}
+              <br />
+              {t('Do you want to apply the change now?')}
+            </TextStyled>
```

Comment:
I think that we can leave it for now, because those texts will be changed during SUDO implementation. We have to think and make those copies as simple as possible without any difficulties that are not needed.

Reply:
ok, that works for me